start_cerberus.py

```diff
@@ -106,9 +106,9 @@ def main(cfg):
         # Cluster info
         logging.info("Fetching cluster info")
         if distribution == "openshift":
-            cluster_version = runcommand.invoke("kubectl get clusterversion")
+            cluster_version = runcommand.invoke("oc get clusterversion")
             logging.info("\n%s" % (cluster_version))
-        cluster_info = runcommand.invoke("kubectl cluster-info | awk 'NR==1' | sed -r "
+        cluster_info = runcommand.invoke("oc cluster-info | awk 'NR==1' | sed "
```

Comment:
@seanogor we might want to use kubectl here since cluster-info resource is common to both kubernetes as well openshift distributions. This way it works even when the distribution is set to kubernetes. Thoughts?